pkg/event/sqlite.go

```diff
@@ -0,0 +1,300 @@
+package event
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/warrant-dev/warrant/pkg/database"
+	"github.com/warrant-dev/warrant/pkg/service"
+)
+
+type SQLiteRepository struct {
+	database.SQLRepository
+}
+
+func NewSQLiteRepository(db *database.SQLite) SQLiteRepository {
+	return SQLiteRepository{
+		database.NewSQLRepository(&db.SQL),
+	}
+}
+
+func (repo SQLiteRepository) TrackResourceEvent(ctx context.Context, resourceEvent ResourceEventModel) error {
+	return repo.TrackResourceEvents(ctx, []ResourceEventModel{resourceEvent})
+}
+
+func (repo SQLiteRepository) TrackResourceEvents(ctx context.Context, models []ResourceEventModel) error {
+	resourceEvents := make([]ResourceEvent, 0)
+	for _, model := range models {
+		resourceEvents = append(resourceEvents, *NewResourceEventFromModel(model))
+	}
+
+	result, err := repo.DB.NamedExecContext(
+		ctx,
+		`
+		   INSERT INTO resourceEvent (
```

Comment:
Now that we've merged #71, you can insert the `id` column in this query as well to use the code generated UUID.